pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -178,6 +178,9 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 					fmt.Sprintf("%s/%s", tr.Namespace, tr.Spec.TaskRef.Name), err),
 			})
 			return nil
```

Comment:
you have a return nil here before the recording the event?

Reply:
good catch. I missed it while resolving conflicts